packages/layout/tasks/build.js

```diff
@@ -54,6 +55,42 @@ async function build() {
         return buildTokenFile(container, 'container');
       },
     },
+    {
+      filepath: path.join(SCSS_DIR, '_size.scss'),
+      builder() {
+        const FILE_BANNER = t.Comment(` Code generated by @carbon/layout. DO NOT EDIT.
+
+ Copyright IBM Corp. 2018, 2019
+
+ This source code is licensed under the Apache-2.0 license found in the
+ LICENSE file in the root directory of this source tree.
+`);
+        const convert = {
+          xSmall: 'xs',
+          small: 'sm',
+          medium: 'md',
+          large: 'lg',
+          xLarge: 'xl',
+          '2XLarge': '2xl',
```

Comment:
```suggestion
          XSmall: 'xs',
          Small: 'sm',
          Medium: 'md',
          Large: 'lg',
          XLarge: 'xl',
          '2XLarge': '2xl',
```

Reply:
I'm still getting `$size-undefined` when running `yarn build`. Changing these to match the capitalization in `index.js` resolved the issue